README.md

```diff
@@ -271,6 +271,18 @@ The last two lines are equivalent as if the variable were substituted for its va
 
 In any expression location, an object containing a single key beginning with "Fn::" calls a built-in function.
 
+##### `Fn::ToBase64`
+
+This function encodes a UTF-8 string with the base64-encoding using the [standard base-64 encoding](https://pkg.go.dev/encoding/base64#pkg-variables). **This will fail if the result is not a valid UTF-8 string**
```

Comment:
```suggestion
Encodes a UTF-8 string with Base64 encoding using the [standard Base64 encoding](https://pkg.go.dev/encoding/base64#pkg-variables). **This will fail if the result is not a valid UTF-8 string**
```

I'm looking at the way other function descriptions are written. They all start with a verb ("Converts", "Calls"...). In the `Fn::FromBase64` description we write "Base64" instead of "base64".

Reply:
I addressed this comment in a new commit. Thanks! Good catch!